tests/ipython/test_ipython.py

```diff
@@ -432,10 +298,10 @@ def test_load_node(
 dummy_input = catalog.load("dummy_input")
 my_input = catalog.load("extra_input")"""
 
-        node_imports = """import package1
-from package2 import module1
-import package3 as pkg3
-import package5.module3"""
+        node_imports = """import logging  # noqa
+from logging import config  # noqa
+import logging as dummy_logging  # noqa
+import logging.config  # noqa Dummy import"""
```

Comment:
@SajidAlamQB I have updated this in the PR and it is included in the PR description. In short, this python module need to be importable. The original code use `package1`,`package2` which are not real package, as a result we need to do many mocking.

I choose `logging` for 2 reasons, 1. it's in standard libraries so no extra dependencies (can be any other standard libraries), 2. It has sub-module so I can test for different kind of import statements. You may swap `logging` with `os` and it will do the same job.